crates/revm/src/chain_spec.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{
+    handler::{ExecutionHandler, PostExecutionHandler, PreExecutionHandler, ValidationHandler},
+    interpreter::opcode::InstructionTables,
+    primitives::{db::Database, spec_to_generic, EthereumWiring},
+    EvmHandler,
+};
+use std::vec::Vec;
+
+pub trait EvmWiring: crate::primitives::EvmWiring {
+    /// The type that contains all context information for the chain's EVM execution.
+    type Context: Default;
```

Comment:
This seems unused, as `EXT` generics were not touched. I am Currently working to include `ExternalContext` and `Database` inside `primitives::EvmWiring` and see how that would look.